src/bpm/runc/adapter/adapter_suite_test.go

```diff
@@ -13,7 +13,7 @@
 // License for the specific language governing permissions and limitations
 // under the License.
 
-package adapter_test
+package adapter
```

Comment:
Was the move from `adapter_test` => `adapter` necessary? Have we removed other `_test` packages in favor of testing from within the package?

Reply:
It wasn't necessary. We were exporting a bunch of variables and functions that were only being used in test, so this felt clean to make the interface of the package what we actually intend it to be. Also as mentioned above, this follows the [standard that golang suggests](https://golang.org/pkg/testing/). The only other package at the moment that follows this pattern is the `mount` package which was added recently.

If the change is too controversial I can back it out, but I didn't find it too disruptive.

Reply:
Mostly just curious. I'm used to the reasoning that tests should be package-external so that only the public interface is exercised... though I realize this isn't a convention most places.